packages/suite/src/components/wallet/WalletLayout/AccountBanners/AccountBanners.tsx

```diff
@@ -24,7 +26,7 @@ export const AccountBanners = ({ account }: AccountBannersProps) => {
     const { route } = useSelector(state => state.router);
 
     return (
-        <>
+        <Column gap={spacings.sm}>
```

Comment:
This adds a gap when there are no banners, since this column is inside another column with gap.

Reply:
Fixed